spec/models/job_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Job, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "#instantiate" do
+    it "should generate new instance" do
+      newclass=described_class.new
+      expect(newclass).to be_a(described_class)
```

Comment:
Done, white spaces cleaned up and one liner added. Please review at your convenient. 
Also you can give me a new feature to work on.